test/app/main/subsystems/json.spec.ts

```diff
@@ -0,0 +1,25 @@
+import {jsonFixtures} from "./json.fixtures"
+import {jsonSerializer} from "../../../../app/common/serializers"
```

Comment:
Here the same, is even shorter to just use `from "app/common/serializers"` :see_no_evil: